### PR TITLE
Adding new docs to upgrade community profile

### LINKED ISSUE #132: Improve CONTRIBUTING.md
# What 

Overhaul the contributing markdown file and/or point it to the dev docs.  Make sure the dev docs have the relevant useful pieces to help external contributors / new contributors.

# Why

Contributing markdown file has lots of useful tips, but it's an old document and is missing a lot of the pieces external contributors would be looking for.

### PR DESCRIPTION
**Pull request by iamkelllly**
# Purpose
Adding and updating existing files to upgrade our community profile

# Changes
* [x] new CONTRIBUTING.md in root directory
* [x] new CODE_OF_CONDUCT.md in root directory
* [x] updated the code of conduct contact email address 

# Checklist

- [ ] Applicable documentation updated (guides, quickstart, postman collections, tutorial, fidesdemo, [database diagram](https://github.com/ethyca/fidesops/blob/main/docs/fidesops/docs/development/update_erd_diagram.md)
- [ ] Good unit test/integration test coverage

# Ticket

Closes #132  and #133

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Fides Contribution Guidelines
```
Comment (pattisdr) [inline CODE_OF_CONDUCT.md:1]: Should say code of conduct instead!

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+The Fides project, which includes Fideslang, Fidesops, and Fidesctl, adheres to the following [Code of Conduct](https://ethyca.github.io/fidesops/community/code_of_conduct/).
```
Comment (pattisdr) [inline CODE_OF_CONDUCT.md:3]: I know we're just directing them to the code of conduct here, but maybe we could rephrase this to be a little more welcoming -

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,223 +1,3 @@
...
-
-Occasionally when developing you'll run into issues where it's beneficial to remove all existing Docker instances in order to recreate them based on some updated spec. Some commands to do this are below:
-
-  - Stop all running containers: `docker-compose down`
-  - Delete all local containers: `docker rm -f $(docker ps -a -q)`
-  - Delete all local Docker volumes: `docker volume rm $(docker volume ls -q)`
-  - Remove temp. files and installed dependencies: `make clean`
-  - Delete all stopped containers, all networks not used by a container, all dangling images, and all build cache: `docker system prune`
-  - Recreate the project: `make compose-build`
+The Fides project, which includes Fideslang, Fidesops, and Fidesctl, adheres to the following [Contribution Guidelines](https://ethyca.github.io/fidesops/development/overview/).
```
Comment (pattisdr) [inline CONTRIBUTING.md:3]: Similar to above i might rephrase to something like Contributions welcome! See <this link> to contribute! something friendlier.

**Review by pattisdr — COMMENTED** [review]
Thank you @iamkelllly! love to see these changes, just a few suggestions -

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (iamkelllly) [reply CODE_OF_CONDUCT.md:3]: Thank you for this! I love being more welcoming but tell me if this update is crossing a line :)

Comment (pattisdr) [reply CODE_OF_CONDUCT.md:3]: I really like this improvement! if I am considering contributing a bugfix to a project, I look at the README first, usually CONTRIBUTING.md second, and sometimes check to see if a code of conduct or similar exists.  An initial perception of friendliness make me more willing to go through the effort.

**Review by pattisdr — APPROVED** [review]
💯
